b2b-app/imports/ui/admin/register/form/form-schemas.js

```diff
@@ -38,45 +48,77 @@ const aboutFormSchema = new SimpleSchema(
       type: String,
       max: 250,
       label: 'Reasons for volunteering',
+      uniforms: {
+        multiline: true,
+        rows: 4,
+        placeholder: [
+          'What makes you want to to volunteer at Back2Bikes?',
+          'Have you ever done any other volunteering before?',
+          'Have you worked on bikes or something similar before?',
+        ].join('\n'),
+      },
     },
   },
   { requiredByDefault: false }
 )
 
 const contactFormSchema = new SimpleSchema({
-  name: String,
-  email: SimpleSchema.RegEx.Email,
+  name: {
+    type: String,
+  },
+  mobileNumber: {
+    type: String,
+    regEx: SimpleSchema.RegEx.Phone,
+  },
+  email: {
+    type: String,
+    regEx: SimpleSchema.RegEx.Email,
+  },
+  pinNumber: {
+    type: String,
+    min: 4,
+    max: 4,
+    uniforms: {
+      'ui:field-cols': 2,
+    },
+  },
+  confirmPIN: {
+    type: String,
+    custom() {
+      if (this.value !== this.field('pinNumber').value) {
```

Comment:
This is nitpicky, but I think having the PIN number input before the address is confusing from a user flow perspective, its going from personal information -> account auth -> personal information

Reply:
Makes sense